x-pack/plugins/infra/public/components/log_stream/log_stream_embeddable.tsx

```diff
@@ -37,7 +37,7 @@ export class LogStreamEmbeddable extends Embeddable<LogStreamEmbeddableInput> {
     initialInput: LogStreamEmbeddableInput,
     parent?: IContainer
   ) {
-    super(initialInput, {}, parent);
+    super(initialInput, { title: initialInput.title ?? 'Log stream' }, parent);
```

Comment:
I wonder if

https://github.com/elastic/kibana/blob/eddf1c94b1c76492e619bef7557a04e8cbbea7d2/src/plugins/embeddable/public/lib/embeddables/embeddable.tsx#L51-L56

doesn't suggest that the correct way to set the initial title would be to set `output.defaultTitle` if at all. Alternatively, could we supply it as `input.title` from the factory?

Reply:
Turns out it wasn't necessary for it to work 🤷‍♂️